interp/modintern.ml

```diff
@@ -104,7 +106,7 @@ let intern_with_decl = function
   | CWith_Module ({CAst.v=fqid},qid) ->
     WithMod (fqid,lookup_module qid)
   | CWith_Definition ({CAst.v=fqid},udecl,c) ->
-    WithDef (fqid,(udecl,c))
+    WithDef (fqid,(c,udecl))
```

Comment:
Why this change? the syntax looks like `with Definition fqid@{udecl} := c`

Reply:
Fixed, thanks!